### ISSUE TITLE
Regarding data copy

### ISSUE THREAD

**Issue by hirotnk** [kvakvs/beam-wisdoms#2]
Hi Dmytro, I think it would be helpful to add info regarding constant pool as described in 8.2 of http://erlang.org/doc/efficiency_guide/processes.html#id69887

Especially where as described as followings:
> But if a constant is sent to another process (or stored in an Ets table), it is copied. The reason is that the runtime system must be able to keep track of all references to constants to unload code containing constants properly. (When the code is unloaded, the constants are copied to the heap of the processes that refer to them.) The copying of constants might be eliminated in a future Erlang/OTP release.

**Comment by kvakvs** [comment]
Even long after release of R20 this document hasn't changed, literals from constant pool will be copied when they leave a process. I vaguely remember that they mentioned some changes to this in R20 but cannot be sure.

**Comment by hirotnk** [comment]
I believe literals are not copied between processes anymore when sent as a message: OTP-13529 in http://erlang.org/download/otp_src_20.0.readme

**Comment by kvakvs** [comment]
Updated http://beam-wisdoms.clau.se/en/latest/eli5-efficiency.html
Closing this, thanks

**Comment by hirotnk** [comment]
Awesome, thank you !
